Buddies/Topics/TopicsVC.swift

```diff
@@ -8,13 +8,17 @@
 
 import UIKit
 import AVFoundation
+import FirebaseAuth
```

Comment:
Do we need to import Auth?

Reply:
We use it for `func loadProfileData(uid: String = Auth.auth().currentUser!.uid, dataAccess: DataAccessor = DataAccessor.instance) -> Canceler {`